quodlibet/quodlibet/ext/events/synchronizedlyrics.py

```diff
@@ -1,6 +1,8 @@
+# -*- coding: utf-8 -*-
 # Synchronized Lyrics: a Quod Libet plugin for showing synchronized lyrics.
 # Copyright (C) 2015 elfalem
 #            2016-17 Nick Boultbee
+# Modified to support mp3 tags and ELRC syntax by Tomás Ralph
```

Comment:
I don't think we add changes to the file. Just year + name (or update year if name already exists).